src/components/appear.js

```diff
@@ -36,11 +36,12 @@ class Appear extends Component {
   render() {
     const child = React.Children.only(this.props.children);
     const endValue = this.state.active ? 1 : 0;
+    const transitionDuration = this.props.transitionDuration || 300;
```

Comment:
I think the proper way would be to use reacts defaultProps to set the transitionDuration, e.g.

```javascript 
Appear.defaultProps = { 
    transitionDuration : 300
};
```